### PR TITLE
always show markdown in blocks mode

### PR DESCRIPTION
**Pull request by pelikhan**
Since the only way to reach that file is through Explorer, it always end up to JavaScript.

### CODE REVIEW

**Comment by samelhusseini** [comment]
Not sure about this one, let me verify when I get in to work

**Comment by pelikhan** [comment]
I searched and I did not see it being used anywhere else.

File: webapp/src/container.tsx [hunk 1]
```diff
@@ -439,7 +439,9 @@ export class SideDocs extends data.Component<SideDocsProps, SideDocsState> {
 
     setMarkdown(md: string) {
         const docsUrl = pxt.webConfig.docsUrl || '/--docs';
-        const mode = this.props.parent.isBlocksEditor() ? "blocks" : "js";
+        // always render blocks by default when sending custom markdown
+        // to side bar
+        const mode = "blocks" // this.props.parent.isBlocksEditor() ? "blocks" : "js";
```
Comment (samelhusseini) [inline webapp/src/container.tsx]: How come?

**Comment by samelhusseini** [comment]
Hmmm. It's used in docs.html, error.html, package.html, script.html, stream.html

File: webapp/src/container.tsx (hunk 1 above)
Comment (pelikhan) [reply webapp/src/container.tsx]: The way to get through Readme.md is through the explorer, which de-facto sets the editor to javascript. However, the setting markdown feature is mostly useful to preview readme.md... which mostly is about seeing blocks.

**Comment by pelikhan** [comment]
Indeed, somehow search is now poping those up. It seems that the sidedocs does not have this "mainbody" id then.

**Comment by pelikhan** [comment]
I'm reporposing this for editing markdown.

**Comment by samelhusseini** [comment]
It's by design that way. I'll keep digging why the docs.less ui.segment CSS is not being applied in this case.

**Comment by samelhusseini** [comment]
Yep, you're right we need a mainbody in webapp/docs.html

**Comment by samelhusseini** [comment]
(Made the change here)

**Pull request merged**